src/main/webapp/resources/js/pages/analysis/components/outputs/OutputFilePreview.jsx

```diff
@@ -20,6 +20,7 @@ import { IconFile } from "../../../../components/icons/Icons";
 const AnalysisTextPreview = React.lazy(() => import("../AnalysisTextPreview"));
 const AnalysisJsonPreview = React.lazy(() => import("../AnalysisJsonPreview"));
 const AnalysisExcelPreview = React.lazy(() => import("../AnalysisExcelPreview"));
+const AnalysisImagePreview = React.lazy(() => import("../AnalysisImagePreview"));
```

Comment:
This file is really get big.  Don't do it know, but if you need to add more to it, refactor some helper functions into their own file.

Reply:
Yeah I thought about doing that as well. If we need to add another one I will break it up. Thanks